index.js

```diff
@@ -30,7 +30,11 @@ app.use((req, res) => {
 
 app.use((err, req, res, next) => { // eslint-disable-line
   //console.log(err);
-  res.status(HSC.INTERNAL_SERVER_ERROR).send(err.message);
+  if (err.status === HSC.FORBIDDEN) {
+    res.status(err.status).send(err.message);
```

Comment:
Maybe it's better to write it like
if () {
return {forbiddenCase}
}

return {defaultCase}

This will be extendable.